src/caustics/lenses/multiplane.py

```diff
@@ -235,6 +396,21 @@ def time_delay(
     ) -> Tensor:
         """
         Compute the time delay of light caused by the lensing.
+        This is based on the work in: McCully et al. 2014 https://ui.adsabs.harvard.edu/abs/2014MNRAS.443.3631M/abstract .
+        For the time delay of a light path from the observer to the source, the following equation is used::
```

Comment:
Probably should cite equation 6.22 of Petters, Levine & Wambsganss 2001 since we take their equation and we don't use the tidal approximation of this expression that is derived in McCully 2014. We can still cite we took the equation from them.